src/excmds.ts

```diff
@@ -1555,18 +1561,20 @@ export function urlparent(count = 1) {
  * @param mode      The replace mode:
  *  * -t text replace
  *  * -r regexp replace
+ *  * -s set the value of the given query
```

Comment:
I don't see any benefit to keeping this separate from `-q`, do you?

Do people often run `-q` to preempt some special query and only replace it if it exists?

Reply:
So I initially thought that the `-q` parameter actually would set the value if it didn't already exist and I was surprised to find that it didn't, nor was there an option to set it if it didn't exist hence this PR.

I can't come up with any particular examples, but I'm sure there exists particular use cases in which some people would want to change the query value only if it exists, whilst not forcing it to always exist.

Reply:
Here's an example I came up with, consider a website which shows a list of things, with the query parameter `limit=10` (default) being used to determine the total number of items to show in the list. You only want to change the `limit=50` when that query exists in the URL and not always forcing the query to exist if it doesn't.

Reply:
In any case, I believe having `-s` and `-q` separate will give more flexibility to the end user without us assuming their typical use case.